console/src/telemetry/Actions.ts

```diff
@@ -130,6 +137,45 @@ const setPreReleaseNotificationOptOutInDB = () => (
   return dispatch(setConsoleOptsInDB(options, successCb, errorCb));
 };
 
+const updateConsoleNotificationsInDB = (updatedState: NotificationsState) => {
```

Comment:
```suggestion
const updateConsoleNotificationsState = (updatedState: NotificationsState) => {
```

Reply:
I thought this would avoid the confusion between the `console_notifications` within the DB and the `consoleNotification` state within `main` in redux